src/site/_collections/posts-with-lighthouse.js

```diff
@@ -14,20 +14,37 @@
  * limitations under the License.
  */
 
-// Return posts that reference a Lighthouse audit.
-// These posts will be displayed in the user's TODO list on the /measure page.
+/**
+ * @fileoverview
+ * Return posts that reference a Lighthouse audit.
+ * These posts will be displayed in the user's TODO list on the /measure page.
+ */
 
 const {livePosts} = require('../_filters/live-posts');
 
-module.exports = (collection) => {
+/**
+ * Filter out posts that aren't related to LH audits.
+ * @param {{data: Object}} post An eleventy post object.
+ * @return {boolean}
+ */
+function hasLighthouseAudit(post) {
+  const audits = post.data.web_lighthouse;
+  if (typeof audits === 'string' && audits !== 'N/A') {
+    return true;
+  }
+  return audits instanceof Array && audits.length !== 0;
+}
+
+/**
+ * Filter all eleventy posts down to a set that are used as LH audit references.
+ * @param {Object} collection An eleventy collection object
+ * @return {Object[]}
+ */
+function postsWithLighthouse(collection) {
   return collection
     .getFilteredByTag('post')
     .filter(livePosts)
-    .filter((post) => {
-      const audits = post.data.web_lighthouse;
-      if (typeof audits === 'string' && audits !== 'N/A') {
-        return true;
-      }
-      return audits instanceof Array && audits.length;
-    });
-};
+    .filter(hasLighthouseAudit);
+}
+
+module.exports = {postsWithLighthouse, hasLighthouseAudit};
```

Comment:
Since this changes the export target, you need to change the imports too, e.g. here: https://github.com/GoogleChrome/web.dev/blob/master/test/integration/src/site/_collections/eleventy.js#L2

I noticed because it broke the test. Yay for integration tests :D